spinta/manifests/tabular/helpers.py

```diff
@@ -1623,6 +1651,12 @@ def datasets_to_tabular(
         else:
             separator = False
 
+        if model.base and (not base or model.base.name != base.name):
+            base = model.base
+            yield from _base_to_tabular(model.base)
+        elif base and not model.base:
+            base = None
+            yield from _base_to_tabular(model.base, end_of_base=True)
```

Comment:
Please add more universal recognition of dimension end marker. End marker should be added for any node, which is not in context of previous node and is in a lower dimension.

For example if model comes after base, resource, dataset and is not in context of previous base, resource and dataset, then before model, and end marker should be added to the last node, that this model is not part of.

For example:

```
dataset
  resource
    base
/
      model
```
Here model does not belong to any previous nodes, so before adding model, and end marker should be added.

Probably a separate function could be added to detect that.

Reply:
Couldn't find universal way to do this, because you need to check different conditions with different dimensions. For now I added option to add "/" to database, resource and base. If this is not a good way, maybe there should be a separate task, because this one already took longer than expected.